docs/export.md

```diff
@@ -5,8 +5,74 @@
 ```
 ## script
 ```javascript
-controller.$inject = ['$scope'];
-function controller($scope){
-	$scope.data = [];
+//export API:
```

Comment:
how this works for the user? how this works for the model? how it will like for grid settings? =)